### ISSUE TITLE
Some OSM object types are missing within "Tag filter"

### ISSUE THREAD

**Issue by iriman** [MarcusWolschon/osmeditor4android#1182]
## Behaviour/Symptoms
No option for Ways+, Relations+ and Relations with Persian locale.

![image](https://user-images.githubusercontent.com/8467903/85334378-681b6d00-b4f0-11ea-8ccb-12428718d020.png)

## Expected Behaviour
All 6 options are available with English locale.

![image](https://user-images.githubusercontent.com/8467903/85334855-2f2fc800-b4f1-11ea-8548-d4e7658d9728.png)

## How to recreate
- Change device locale to Persian.
- Enable tag filter. 
- try to choose ways+ in filter dialog

## Crash dump submitted (no or yes + date)
no

## Vespucci Version
14.1.4 Persian locale

## Download source
F-Droid

## Device (Manufacturer and Model)
Samsung SM-J510F

## Android Version 
7.1.1

**Comment by simonpoole** [comment]
Hmm, I don't have an explanation for that right now. The translations are present and it works on both 7.1 and 9.0 without issues, I also didn't see a partial translation anywhere. 

I would suggest trying 14.1.5 (which should be hitting the play store rsn) and see if that resolves the issue.

**Comment by simonpoole** [comment]
Seems as if the translation updates didn't work correctly for the 14.1 branch, fixed now and will be available in 14.1.6

**Comment by iriman** [comment]
> I would suggest trying 14.1.5 (which should be hitting the play store rsn) and see if that resolves the issue.

I uninstalled it and reinstalled using play store (however I didn't understand what is rsn). It was 14.1.4 available on play store for me. I also joined the beta test program that there was no other version to be installed.

I test again with 14.1.4 from play store and same issue happened. But I did one more thing:

I changed locale to English and then applied a filter with Ways+ enabled. Then while filter was on, switched device locale to Persian and chose the Vespucci from recent app list (or re-ran it), which raised a crash. I sent crash log around 10:00 PM 22-06-2020. Hope it has something useful.

> Seems as if the translation updates didn't work correctly for the 14.1 branch, fixed now and will be available in 14.1.6

Thanks
